scripts/deploy.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# set -x
+set -eo pipefail
+
+# --- GLOBAL CONSTANTS
+
+BETTING_PERIOD_LENGTH=5
+MAXIMAL_NUMBER_OF_BETS=5
+MINIMAL_BET_AMOUNT=1000000000000
+
+INK_DEV_IMAGE=public.ecr.aws/p6e8q1z1/ink-dev:1.5.0
+NODE=ws://127.0.0.1:9944
+AUTHORITY=5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
+AUTHORITY_SEED=//Alice
```

Comment:
What about using drink for these tests? We could be the first repo to consume its testing library features. Then we don't have to assume anything about account id at the contract level.

cc @pmikolajczyk41

Reply:
I've just prepared some utilities to test things like this: https://github.com/Cardinal-Cryptography/drink/blob/main/examples/flipper/lib.rs#L34

if it is sufficient for this case (I've not reviewed this PR yet), then go on :)